src/containers/Page.js

```diff
@@ -1,15 +1,14 @@
 import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
 import BattleArena from './Game/BattleArena'
 import MenuBar from './Game/MenuBar'
+import {connect} from 'react-redux'
+import {Redirect} from 'react-router-dom'
+import {getGames, joinGame, updateGame} from '../actions/games'
+import {getUsers} from '../actions/users'
+import {userId} from '../jwt'
 
-
-//STYLING
-
-
-//CLASSES
-
-
-export default class Page extends PureComponent {
+export class Page extends PureComponent {
```

Comment:
What page is this?

Reply:
temporarily name. should be gameScreen or something.